compiler/tf2tfliteV2/tf2tfliteV2.py

```diff
@@ -15,7 +15,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import tensorflow as tf
+import os
+
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
```

Comment:
I made a draft patch for verbose option. (#7303)
In this PR, I used the publicly log environment. I thought it would be easier to implement. Would it be better to have different log environment for each tools? 

```
def _set_verbose_env(verbose_level):
    """set verbose environment"""
    os.environ['ONE_VERBOSE'] = str(verbose_level)
```

Reply:
There are many things in `one-cmds`: `onecc`, `one-build`, `one-xxx` other than `one-build`, and its implementations like `tf2tfliteV2.py`. Do you want to control verbose level all tools above by `ONE-VERBOSE` environment variable? It would be the easiest way that could be done in time. However, I have several questions. Let's talk offline soon.

Reply:
> Do you want to control verbose level all tools above by ONE-VERBOSE environment variable?

No, each module can use its own log environment. However, it would be better to manage each log env through the entire log env.